src/resources/js/views/datepicker.js

```diff
@@ -225,7 +370,12 @@ tribe.events.views.datepicker = {};
 		var $datepickerButton = $container.find( obj.selectors.button );
 		var viewSlug = data.slug;
 		var isMonthView = 'month' === viewSlug;
+		var isWeekView = 'week' === viewSlug;
 		var minViewMode = isMonthView ? 'year' : 'month';
+		/**
+		 * @todo: use format from BE
```

Comment:
Reminder to assign a name even if for yourself ;)

Reply:
thanks, did this before our call and didn't update 🤦‍♂